models/public/yolof/README.md

```diff
@@ -0,0 +1,124 @@
+# yolof
+
+## Use Case and High-Level Description
+
+YOLOF is a simple, fast, and efficient object detector without FPN. Model based on ["You Only Look One-level Feature"](https://arxiv.org/abs/2103.09460) paper. It was implemented in PyTorch\* framework. Model used `CSPDarkNet-53` as backbone. For details see [repository](https://github.com/megvii-model/YOLOF). This model was pre-trained on [Common Objects in Context (COCO)](https://cocodataset.org/#home) dataset with 80 classes.
```

Comment:
```suggestion
YOLOF is a simple, fast, and efficient object detector without FPN. Model based on ["You Only Look One-level Feature"](https://arxiv.org/abs/2103.09460) paper. It was implemented in PyTorch\* framework. Model used `DarkNet-53` with Cross Stage Partial blocks as backbone. For details see [repository](https://github.com/megvii-model/YOLOF). This model was pre-trained on [Common Objects in Context (COCO)](https://cocodataset.org/#home) dataset with 80 classes.
```

Reply:
better, but still unclear which model used  from these 3:
<html>
<body>
<!--StartFragment-->

YOLOF_CSP_D_53_DC5_3x | 41.2 | 39
-- | -- | --
YOLOF_CSP_D_53_DC5_9x | 42.8 | 39
YOLOF_CSP_D_53_DC5_9x_stage2_3x | 43.2 | 39

<!--EndFragment-->
</body>
</html>

Reply:
As far as I understood, this is one architecture, but pre-trained with different hyperparameters (steps, ...). So what info we can demonstrate for customers?